install/cupy_builder/install_build.py

```diff
@@ -16,6 +18,8 @@
 from cupy_builder._context import Context
 
 
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Question: Is `logging` mandatory? It's better if we can avoid mutating global states because this file is called from unit tests.

Reply:
Yes I can hide it under the env var `CONDA_BUILD`.

Reply:
Thanks, then could you put it under `CONDA_BUILD` check (or just replace the logging with `print` also work?)

Reply:
Done in commit [35b836b](https://github.com/cupy/cupy/pull/7603/commits/35b836b8ca16211b4e504f587730cc39e1e2159e).